rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java

```diff
@@ -110,6 +110,9 @@ public MockPreparedStatement createSeataMockPreparedStatement(MockConnection con
      * @param mockReturnValue
      */
     public void setMockReturnValue(Object[][] mockReturnValue) {
+        if (mockReturnValue == null) {
+            this.mockReturnValue = new Object[][]{};
```

Comment:
Need return here, change `if ... else ...` or `... ? ... : ...` could be better

Reply:
Fixed. Thanks